pkgs/by-name/ho/hoppscotch-community-edition/package.nix

```diff
@@ -0,0 +1,122 @@
+{
+  fetchFromGitHub,
+  lib,
+  makeBinaryWrapper,
+  nodejs,
+  pnpm,
+  prisma-engines,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "hoppscotch-community-edition";
+  version = "2024.8.1";
+
+  src = fetchFromGitHub {
+    owner = "hoppscotch";
+    repo = "hoppscotch";
+    rev = finalAttrs.version;
+    hash = "sha256-qXBiL573n4iX4KQ86UL1eGV2Ox9hlGZe/edF10f11PM=";
+  };
+
+  prePnpmInstall = ''
+    pnpm config set dedupe-peer-dependants false
+  '';
+  pnpmDeps = pnpm.fetchDeps {
+    inherit (finalAttrs)
+      pname
+      version
+      src
+      prePnpmInstall
+      ;
+    hash = "sha256-9J15k1ZSU4UBuXNN/b+vwRKNsk2MgrTXMRLyYX9TtIg=";
```

Comment:
Please rebase and regenerate hashes.

See https://github.com/NixOS/nixpkgs/pull/349093#issuecomment-2421873637